### ISSUE TITLE
Uncaught (in promise) TypeError: this.positionCleanup is not a function

### ISSUE THREAD

**Issue by peremarshall** [joeattardi/picmo#206]
This API is working great, but on emoji selection or clicking outside of the popup window, the following error is consistently thrown...

`Uncaught (in promise) TypeError: this.positionCleanup is not a function`

I'm sourcing the library using the <script> tags...

```
<script src="https://unpkg.com/picmo@5.1.0/dist/umd/picmo.js"></script>
<script src="https://unpkg.com/@picmo/popup-picker@5.1.0/dist/umd/picmo-popup.js"></script>
```

My code is as follows...

```
  var emojiPickerTrigger = document.querySelector('.emoji-input');

  if (emojiPickerTrigger) {
    var emojiPicker = picmoPopup.createPopup({}, {
      triggerElement: emojiPickerTrigger,
      position: { bottom: "10px" }
    });

    emojiPickerTrigger.addEventListener("click", function() {
      emojiPicker.toggle();
    });

    emojiPickerTrigger.addEventListener('emoji:select', function(event) {
      document.querySelector('.emoji-input').value = event.emoji
    });
  }
```

Am I missing a key source package or something?

**Comment by peremarshall** [comment]
Oops... I was obviously using the position attribute incorrectly. I still don't really don't understand how to used FixedPosition correctly, but that's on me. 😊

**Comment by joeattardi** [comment]
Actually, looks like this is a valid issue! If fixed positioning is used, then no positionCleanup gets added (currently it only gets added if a reference element is passed in, which you wouldn't do if you are using fixed positioning). But upon closing the picker, as you found, it tries to run the cleanup, hence the exception.

I'll make sure to address this soon.

As for the difference, when you use fixed positioning, the picker is fixed to the viewport. It won't scroll if the page is scrolled. You can specify left/right/top/bottom position values for it.

Relative positioning takes a reference element, and the picker is positioned relative to that. It will scroll with the container.

See [here](https://picmojs.com/docs/usage/creating-pickers#positioning-the-popup).

**Comment by joeattardi** [comment]
Just released 5.1.1 which should solve the issue with the exception thrown with `positionCleanup` when using fixed positioning.

**Comment by peremarshall** [comment]
Awesome! Thanks.
